.pylintrc

```diff
@@ -7,7 +7,8 @@
 # W0703: broad-except - We are dealing with many different source systems, but we want to make sure workflows run until the end
 # W0511: fixme - These are internal notes and guides
 # W1518: method-cache-max-size-none - allow us to use LRU Cache with maxsize `None` to speed up certain calls
-disable=W1203,W1202,R0903,W0707,R0901,W1201,W0703,W0511,W1518
+# C0412: ungrouped-imports - Since we have namespace packages which are grouped differently by pycln. Better to have this automated and unchecked.
+disable=W1203,W1202,R0903,W0707,R0901,W1201,W0703,W0511,W1518,C0412
```

Comment:
removing import order flag from pylint - let's stick to `pycln` results from automation